src/memory/title_sequence_manager.rs

```diff
@@ -29,104 +47,352 @@ impl MemoryManagerUpdate for TitleSequenceManagerData {
         ctx: &StateContext,
         manager: &mut UnityMemoryManager,
     ) -> Result<(), Error> {
-        match self.update_title_menu(ctx, manager) {
-            Ok(_) => Ok(()),
-            Err(error) => Err(error),
+        if let Some(class) = manager.class {
+            if let Some(process) = &ctx.process {
+                if let Some(module) = &ctx.module {
+                    if let Some(singleton) = manager.singleton {
+                        self.update_current_screen_name(class, process, module, singleton)?;
+                        self.update_title_menu(class, process, module, singleton)?;
+                        self.update_pressed_start(class, process, module, singleton)?;
+                        self.update_load_save_done(class, process, module, singleton)?;
+                        self.update_relics(class, process, module, singleton)?;
+                        self.update_new_game_characters(class, process, module, singleton)?;
+                    }
+                }
+            }
         }
+        Ok(())
```

Comment:
Atm this means that there is no Error that can be returned from this function, which maybe intended (it silently fails).

Reply:
any `?` function should propogate out, so yea this just returns true if there was no errors on the Results

https://doc.rust-lang.org/rust-by-example/std/result/question_mark.html

However if we errored on the if let bindings of process/module/singleton, the managers would reset indefinitely - This is something i'm going to be tidying up for performance after we have a stable setup.

What would need to happen is similar to the other tas that managers should only be updated when they are needed specifically, so they dont constantly reset when the manager doesn't exist